lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -39,13 +39,14 @@ def change
       # https://github.com/doorkeeper-gem/doorkeeper/tree/v3.0.0.rc1#custom-access-token-generator
       #
       # t.text     :token,             null: false
-      t.string   :token,             null: false
+      t.string   :token,                  null: false
 
       t.string   :refresh_token
       t.integer  :expires_in
       t.datetime :revoked_at
-      t.datetime :created_at,        null: false
+      t.datetime :created_at,             null: false
       t.string   :scopes
+      t.string   :previous_refresh_token, null: false, default: ""
```

Comment:
Should we add a comment here, prompting developers to comment out this column if they wish to revoke tokens instantly, as it was before? Something like:

`# If you have a `previous_refresh_token` column, refresh tokens will be revoked only after a generated access token is used. If you don't, it will be revoked as soon as a new access token is issued.`